### PR TITLE
Backport Admin Technical Features to 7.0

### PR DESCRIPTION
**Pull request by bwrsandman**

### CODE REVIEW

File: admin_technical_features/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    'name': 'Admin Technical Features',
+    'version': '0.1',
+    'author': 'Savoir-faire Linux',
+    'maintainer': 'Savoir-faire Linux',
+    'website': 'http://www.savoirfairelinux.com',
+    'license': 'AGPL-3',
+    'category': 'Administration',
+    'summary': 'Checks the technical features box for admin user.',
+    'description': """
+Base Partner Compute Field
```
Comment (pedrobaeza) [inline admin_technical_features/__openerp__.py]: Use OCA's template

**Comment by pedrobaeza** [comment]
The icon is already used in another OCA's module: https://github.com/OCA/web/blob/8.0/web_tree_many2one_clickable/static/description/icon.png and it's not very descriptive. Do you think it deserved to have it instead of going of OCA's standard one?

**Comment by bwrsandman** [comment]
@nhomar What is the in progess tag for?

**Comment by bwrsandman** [comment]
@pedrobaeza this is a backport, it is not meant to much change from the original

> The icon is already used in another OCA's module

It is? Well, this icon predates the one you linked and the module predates yours by four months:
https://github.com/OCA/server-tools/commit/cb348f39ad8441d8cf99669293268b06f5bd6133

Following that logic, web_tree_many2one_clickable should change its icon, not this backport.

File: admin_technical_features/__openerp__.py (hunk 1 above)
Comment (bwrsandman) [inline admin_technical_features/__openerp__.py]: We would like it to follow the best practices, could you link the best practices page so I may make the changes?

**Comment by pedrobaeza** [comment]
The changes I mention are only style changes. About the icon, I mention because now you're making the PR, but you can do what you want about it. It's only a suggestion, because the icon doesn't say anything about the functionality, and I prefer the aesthetic OCA's default one.

File: admin_technical_features/__openerp__.py (hunk 1 above)
Comment (pedrobaeza) [inline admin_technical_features/__openerp__.py]: I don't know if this is written in any place, but as you are an usual in OCA's reviews, I though that you already know the OCA's README template: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst, that it was mainly pushed by your company.

**Comment by bwrsandman** [comment]
@pedrobaeza 
Changes to description done! Forward port of the fix to the backport, please review: #165

**Comment by pedrobaeza** [comment]
Thanks :+1:

**Comment by lmignon** [comment]
Thank you :+1:  (Code review)

**Pull request merged**